scripts/e2e/delete_fixtures

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -eu
+
+# Does the inverse of create_fixtures
+echo && echo "[$0] deleting fixtures for api e2e tests..."
+
+delete_doc() {
+	local docUrl="$COUCH_URL/$1"
+	local rev="$(curl $docUrl | jq --raw-output ._rev)"
```

Comment:
oooh you can quote within quotes :woman_facepalming: 
I assumed you couldn't, and was trying to escape them, and so curl was complaining about `""http"`...

Thanks! Done.